int32 id
---
bool setforce_clb_accepted
